SimpleWAWS/Controllers/TemplatesController.cs

```diff
@@ -39,5 +46,18 @@ public IEnumerable<BaseTemplate> Get()
             list.Add(WebsiteTemplate.EmptySiteTemplate);
             return list;
         }
+        public HttpResponseMessage GetARMTemplate(string templateName)
+        {
+            var list = TemplatesManager.GetTemplates().ToList();
```

Comment:
You don't really need .ToList(). I thought GetTemplates() already return an IEnumerable